packages/discord.js/test/sendtest.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
+const fetch = require('node-fetch');
```

Comment:
Shouldn’t node imports stay before everything else and not be sorted alphabetically like this was?

Reply:
Don't care much about those test files regardless.

Reply:
Ah true, I didn’t realize it was a test file